src/maps

```diff
@@ -512,9 +539,31 @@ def mode_run(repo, args):
         print("Sandbox exited successfully!")
 
 
-def download(repopath, remote, target):
-    """Function to download a repo from remote"""
+def telemetry_ping(repo, remote: str, target: str):
+    """
+    Function to transmit telemetry about what runtime is about to be downloaded to server.
+    This happens only if the user has opted into providing telemetry.
+    """
+    # strip password data from remote
+    remotes = list_remotes(repo)
+    o = urlparse(remotes[remote])
+    newo = f"{o.scheme}://"
+    newo += o.netloc.split('@')[1] + o.path
```

Comment:
So an example would be `https://username:password@server.url/path` and we want to strip out the part before the `@`. I'd suggest to include this kind of example as a comment...

As yo just suggested:
```suggestion
    newo += o.netloc.split('@')[-1] + o.path
```